GoogleForCreators/web-stories-wp
# Blocks: Usage difficulties due to theme styling

## Summary

The align functionality of blocks, using floats to align right / left. Right and left float can result in issues with 0 heights. 

## Relevant Technical Choices

- In the CMS, the float without a fixed, means that the block itself results in 0 height, meaning it can not be selected. Adding a simple clearfix, seems to workaround this issue. 
- Add margin-left/right to align left/right. 

## To-do



## User-facing changes



## Testing Instructions
1. Click Themes 
2. Activate Twenty sixteen theme. 
3. Create a new post. 
4. Create story embed block. 
5. Input url. 
6. Align right or left. 
7. Save post. 
8. View post on FE. 

This should be tested again all core themes. 



---



Fixes #5239
